exasol/nb_connector/itde_manager.py

```diff
@@ -144,61 +154,61 @@ def _get_ipv4_addresses():
     return ip_addresses
 
 
-def is_itde_running(conf: Secrets) -> Tuple[bool, bool]:
+def get_itde_status(conf: Secrets) -> ItdeContainerStatus:
```

Comment:
```suggestion
def itde_status(conf: Secrets) -> ItdeContainerStatus:
```

Reply:
I think it's better to have a verb in the name.